admin/src/collections/recipients/RecipientsProperties.ts

```diff
@@ -124,7 +124,9 @@ export const organisationProperty: Property = {
 export const emailProperty: Property = {
 	name: 'Email',
 	dataType: 'string',
+	email: true,
 	hideFromCollection: true,
+	validation: {},
```

Comment:
Let's set this to 

```ts
	validation: { unique: true },
``` 

and then I'll merge, ok?

Reply:
Yeah that totes makes sense, will do 👌🏾